packages/ckeditor5-dev-build-tools/src/plugins/splitCss.ts

```diff
@@ -30,10 +30,35 @@ export interface RollupSplitCssOptions {
  */
 const filter = createFilter( [ '**/*.css' ] );
 
-/**
- * Regular expression to match CSS variables.
- */
-const VARIABLE_DEFINITION_REGEXP = /--([\w-]+)/gm;
+const commonPurgeCssOptions = {
+	fontFace: true,
+	keyframes: true,
+	variables: true
+};
+
+const purgeCssOptionsForContent = {
+	...commonPurgeCssOptions,
+	content: [],
+	safelist: { deep: [ /^ck-content/ ] },
+	blocklist: []
+};
+
+const purgeCssOptionsForEditor = {
+	...commonPurgeCssOptions,
+	// Pseudo class`:where` is preserved only if the appropriate html structure matches the CSS selector.
+	// It's a temporary solution to avoid removing selectors for Show blocks styles where `:where` occurs.
+	// See: https://github.com/FullHuman/purgecss/issues/978
```

Comment:
I'd be nice to know what **exact** issue this HTML solves.

Reply:
Addressed in 96d3bb7.